app/views/search/advanced/index.html.erb

```diff
@@ -14,38 +14,25 @@
 
           <div class="form-group-search row">
             <div class="col col-form-label-lg">
-              <%= label_tag 'City' %>
-              <%= text_field_tag :city, nil, placeholder: 'Search by City', class: "form-control" %>
-            </div>
-          </div>
-
-          <div class="form-group-search row">
```

Comment:
instead of city and state we are now just searching by location with a text field

Reply:
Nice, this will definitely make things simpler.